Cargo.toml

```diff
@@ -8,6 +8,7 @@ edition = "2018"
 aes-gcm = "0.9"
 async-trait = "0.1"
 async-recursion = "0.3"
+backoff = { version = "0.3.0", features = ["tokio"] }
```

Comment:
I'm not sure if we're still making use of this?

Reply:
Removed